episodes/chapter6.md

```diff
@@ -0,0 +1,570 @@
+2024-05-24 15:52:13 - context        - INFO     - Logging to '/Users/clgeng/Library/Logs/vantage6/node/node1/node_user.log'
+2024-05-24 15:52:13 - context        - INFO     - Common package version '4.1.0'
+2024-05-24 15:52:13 - context        - INFO     - vantage6 version '4.1.0'
+[info ] - Configuration file = ***/vantage6/node/node1.yaml
+[info ] - Log file           = ***/vantage6/node/node1/node_user.log
+[info ] - data folders       = ***/vantage6/node/node1
+[info ] - Database labels and files
+[info ] -  - default         = ***/data.csv (type: csv)
+[info ] -  - default         = ***/data.sql (type: sql)
+```
+
+You can see that not only the path of the configuration file is printed out, but also the locations of the log file, the data folders and the database files are shown.
+
+::::::::::::::::::::::::::::::::::::: challenge
+
+## Challenge 2: Create a new node configuration
+
+1. Create a new node configuration using the `v6 node new` command.
+2. Find the path to the configuration file using the `v6 node files` command. Open the configuration file with a text editor and check the configuration options. Are they correct?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Start a node
+
+Before starting a vanatge6 node, you need to make sure the vantange6 server is running and the Internet connection is stable.
+
+To start a node, you can run the command `v6 node start`,
+
+```bash
+v6 node start
+```
+
+It will ask you which node you want to start. You can choose the one you just created,
+
+```bash
+[info ] - Starting node...
+[info ] - Finding Docker daemon
+? Select the configuration you want to use: (Use arrow keys)
+ » node1
+   node2
+   node3
+```
+
+then it will start the node and print out the following messages,
+
+```bash
+? Select the configuration you want to use: node1
+[info ] - Starting node...
+[info ] - Finding Docker daemon
+[info ] - Checking that data and log dirs exist
+[info ] - Connecting to server at 'http://localhost:5000/api'
+[info ] - Pulling latest node image 'harbor2.vantage6.ai/infrastructure/node:4.5'
+[info ] - Creating file & folder mounts
+[warn ] - private key file provided ***/private_key.pem, but does not exists
+[info ] - Setting up databases
+[info ] -   Processing csv database default: ***/data.csv
+[debug] -   - non file-based database added
+[info ] -   Processing csv database default: ***/data.csv
+[debug] -   - non file-based database added
+[info ] - Running Docker container
+[info ] - Node container was successfully started!
+[info ] - To see the logs, run: v6 node attach --name node1
+```
+
+🎉 Now, the node is started successfully!
+
+
+If you see a warning message like this,
+```bash
+[warn ] - Version mismatch between CLI and server/node. CLI is running on version 4.1.0, while node and server are on version 4.5. This might cause unexpected issues; changing to 4.5.<latest> is recommended.
+```
```

Comment:
I don't expect this to happen in the workshop as they will have a fresh installation. So maybe better to leave it out